server/api.go

```diff
@@ -155,7 +155,7 @@ func checkPluginRequest(next http.HandlerFunc) http.HandlerFunc {
 	}
 }
 
-func (p *Plugin) ServeHTTP(c *plugin.Context, w http.ResponseWriter, r *http.Request) {
+func (p *Plugin) ServeHTTP(_ *plugin.Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
Do we want to tune the linter to not warn about this stuff? It doesn't seem helpful to me.

Reply:
@hanzei Sure, this has caused the same problem in every other plugin. Can this be a centralized change, or does this need to be applied to each repo individually?

Reply:
I don't think we can fix this per repo.

Reply:
@hanzei Would you happen to know which repo the fix goes into? Possibly `actions`?

Reply:
Sorry, my previous comment was ambiguous. I meant that we must update `.golangci.yml` for every repository.